infra/localnightly.sh

```diff
@@ -0,0 +1,3 @@
+# pass arguments to nightly.sh
```

Comment:
Why does this need to be a separate script? Or, can it be python instead of bash?

Reply:
We would bounce from python to bash and back to python again, since the nightly script is in bash
Lets keep this in mind if this script gets any more complex

Reply:
Alternatively, why can't this be in the make file like it was previously? I'm not sure I understand wanting to throw this small thing into a bash script.

I think a `make serve-nightly` might be better to run only the server, but am open to whatever

Reply:
It takes an argument, which make doesn't support ):

Reply:
Ahhhhhh, I think I see why it is an issue. Is there a reason it can't be an env var instead of an argument?

Reply:
reason being I really don't like env vars

Reply:
I personally like the idea of a default variable inside of the make file that'll run the local nightlies with everything, but then you can overwrite it with an env var